content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -9,35 +9,6 @@ weight: 90
  from the community. Please try it out and give us feedback!
 {{< /caution >}}
 
-## kubeadm alpha kubeconfig user {#cmd-phase-kubeconfig}
```

Comment:
because `certs` is also deprecated in 1.21.

What do you think like this？

(well there are `kubeconfig` and `certs` but there are deprecated)

Reply:
instead of the removed commands 

```
## kubeadm alpha kubeconfig user {#cmd-phase-kubeconfig}
...
## kubeadm alpha kubelet config {#cmd-phase-kubelet}
...
## kubeadm alpha selfhosting pivot {#cmd-selfhosting}
...
```

you can just add:
> Currently there are no experimental commands under kubeadm alpha.


the only command in 1.21 will be `kubeconfig`:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/alpha/kubeconfig.go

but it's deprecated and Cobra docs will not be generated for it.